### ISSUE TITLE
Exim4 relay rejected for ipv6 host

### ISSUE THREAD

**Issue by MikeDombo** [tianon/dockerfiles#339]
https://github.com/tianon/dockerfiles/blob/master/exim4/Dockerfile#L29
^ Code seems to imply that only ipv4 hosts will be allowed to relay, though in the same file, it is listening on both ipv4 and ipv6.

**Comment by tianon** [comment]
Good catch -- Docker networks don't typically have IPv6 enabled, so this
doesn't really make much difference for any of the ways I deploy this, but
is probably something worth fixing.
